core/standard/requirements/crawlable-catalog/REQ_catalog-collection-itemType.adoc

```diff
@@ -0,0 +1,10 @@
+[[req_crawlable-catalog_collection_itemType]]
+[width="90%",cols="2,6a"]
+|===
+^|*Requirement {counter:req-id}* |*/req/crawlable-catalog/catalog-collection-itemType*
+
+^|A |If a catalog collection object homogenously references records then, its `itemType` member SHALL be a string with the fixed value of `record`.
```

Comment:
^|A |If a catalog collection object homogeneously references records, then its `itemType` member SHALL be a string with the fixed value of `record`.

Reply:
Fixed.